src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAccountController.cs

```diff
@@ -136,25 +136,47 @@ public async Task<ActionResult> GateWayResponse()
         }
 
         [HttpGet]
-        [Route("youhaveregistered")]
-        public ViewResult YouHaveRegistered(string hashedAccountId = null)
+        [Route("getGovernmentFunding")]
+        public ActionResult GetGovernmentFunding()
         {
-            var cookie = _employerAccountOrchestrator.GetCookieData();
-
-            if (cookie == null)
+            var model = new
             {
-                ViewBag.RequiresPayeScheme = true;
-            }
-            else
+                HideHeaderSignInLink = true
+            };
+
+            return View(model);
+        }
+
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        [Route("getGovernmentFunding")]
+        public async Task<ActionResult> GetGovernmentFunding(int? choice)
+        {
+            switch (choice ?? 0)
             {
-                ViewBag.RequiresPayeScheme = string.IsNullOrEmpty(cookie.EmployerAccountPayeRefData.PayeReference) ||
-                                             cookie.EmployerAccountPayeRefData.EmpRefNotFound;
-            }
+                case 1:
```

Comment:
Can the numbers be changed to an enum or constants so it's obvious what they're for?